.env.juno.example

```diff
@@ -11,6 +11,9 @@ SIGN_OF_LIFE="30"  #Sign of Life in Minutes. E.g. "30"
 # SLACK_TOKEN = 
 # SLACK_CHANNEL = 
 # DISCORD_WEBHOOK_URL = 
+# TELEGRAM_CHAT_ID =
+# TELEGRAM_BOT_TOKEN =
+EXTERNAL_EXEMPT_CODES="4, 5, 6, 8"
```

Comment:
what are those EXTERNAL_EXEMPT_CODES for exactly?

> Codes that are not sent to external sources (discord, slack).

Are those Code ID of SmartContracts?

Reply:
they are errorcodes returned by skip that one does not want to see on the external loggers, but only on the stdout